rpm/camera-settings-plugin.spec

```diff
@@ -47,8 +47,8 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%{_libdir}/libonyxcamerasettings-qt5.*
-%{_libdir}/qt5/qml/com/kimmoli/onyxcamerasettings/
+%{_libdir}/libcamerasettings-qt5.*
+%{_libdir}/qt5/qml/com/kimmoli/camerasettings/
 /usr/share/jolla-settings/entries/
-/usr/share/jolla-settings/pages/onyx-camera-settings/
+/usr/share/jolla-settings/pages/camera-settings-plugin/
```

Comment:
extra -plugin here.
will fail as entry points to "path": "system_settings/system/camera-settings",